build.gradle

```diff
@@ -2,6 +2,20 @@ import org.gradle.internal.os.OperatingSystem
 import org.jabref.build.xjc.XjcPlugin
 import org.jabref.build.xjc.XjcTask
 
+// custom build script for spotbugs integration with gradle
+// see https://plugins.gradle.org/plugin/com.github.spotbugs
+// further development will see this refactored to gradle settings
```

Comment:
This line is not clear. Is it "Currently, it is not possible to include this as plugin?`? - But I see it as plugin below.

Maybe, this whole `buildscript` is not necessary, because spotbugs is contained in the plugin section?

Reply:
IDK why gradle still lists the legacy configuration. This is now years old...

Spotugs fails if an issue was found. If that is confusing for you during setup, change `ignoreFailures` to `true` in `build.gradle`.